storage/rocksdb/CMakeLists.txt

```diff
@@ -34,6 +34,15 @@ IF ((CMAKE_CXX_COMPILER_ID STREQUAL GNU) AND
   RETURN ()
 ENDIF ()
 
+# Relax fallthrough warnings we have no control over, as it's in zstd
```

Comment:
@georgelorchpercona, we need to remember to remove this once the upstream fixes those warnings